uicomponent-compose/main/src/main/java/io/github/droidkaigi/feeder/AppContent.kt

```diff
@@ -35,16 +36,27 @@ import kotlinx.coroutines.launch
 private const val FEED_PATH = "feed/"
 private const val OTHER_PATH = "other/"
 
+private val drawerOpenedStatusBarColor = Color.Black.copy(alpha = 0.48f)
+
 @Composable
 fun AppContent(
     modifier: Modifier = Modifier,
     firstDrawerValue: DrawerValue = DrawerValue.Closed,
 ) {
-    val drawerState = rememberDrawerState(firstDrawerValue)
+    val systemUiController = rememberSystemUiController()
+    val drawerState = rememberDrawerState(firstDrawerValue) {
+        if (it == DrawerValue.Closed) {
+            systemUiController.setStatusBarColor(Color.Transparent)
+        } else {
+            systemUiController.setStatusBarColor(drawerOpenedStatusBarColor)
+        }
+        true
+    }
```

Comment:
How about doing this? I think that this can erase the processing using systemUiController of onNavigationIconClick and DrawerContent.

```kotlin
    val drawerState = rememberDrawerState(firstDrawerValue)
    LaunchedEffect(drawerState.currentValue) {
        if (drawerState.currentValue == DrawerValue.Closed) {
            systemUiController.setStatusBarColor(Color.Transparent)
        } else {
            systemUiController.setStatusBarColor(drawerOpenedStatusBarColor)
        }
    }
```

Reply:
Thank you for your comment!
I'll check it over the weekend.

Reply:
In 8dc84b7 commit uses `LaunchedEffect`  and it works.
Additionally, bug that status bar color does not change when drawer is closed by touching outside drawer is fixed :+1: